src/libraries/System.Memory/tests/Span/StringSearchValues.cs

```diff
@@ -469,6 +470,254 @@ public static void TestIndexOfAny_RandomInputs()
             helper.TestRandomInputs();
         }
 
+        [Fact]
+        public static void IsWhiteSpace_True()
```

Comment:
Please move these tests to https://github.com/dotnet/runtime/tree/main/src/libraries/System.Memory/tests/ReadOnlySpan

`SearchValues` is an implementation detail, and this file is specifically focused on `SearchValues<string>`, which we aren't using here.